noxfile.py

```diff
@@ -125,7 +125,7 @@ def test(session, pandas_version: str):
         "-m",
         "pytest",
         "--cov-report=term-missing",
-        "--cov=opensearch_py_ml/",
+        "--cov",
```

Comment:
what does it do now? do you see any other concern to change like this?

Reply:
By changing it like that, it allows us to specify in the setup.cfg instead what files we want to cover.

<img width="593" alt="Screenshot 2023-08-21 at 1 33 56 PM" src="https://github.com/opensearch-project/opensearch-py-ml/assets/106889996/4da77789-21f9-4fb1-a349-ace6d7f920d1">